src/dune_engine/string_with_vars.ml

```diff
@@ -83,6 +83,42 @@ let decode_manually f =
 
 let decode = decode_manually Pform.Env.parse
 
+let decode_many f =
```

Comment:
I don't see this function being used anywhere?

Reply:
good catch, was from a previous implementation attempt.